azerothcore/azerothcore-wotlk
# fix(DB/npc_vendor): Argent Tournament

## CHANGES PROPOSED:
This PR fixes/improve all NPC vendors in argent tournament area (Horde & Alliance)

-  Add all `slot` values for each vendor so these are not sorted by the id. So it fixes the sequence of the items (Now they are all the same = blizzlike)
- Remove duplicated items like:

![2020-12-04 16_29_16-World of Warcraft](https://user-images.githubusercontent.com/55922592/101213729-2552e900-367b-11eb-92e5-67537378582f.png)



## ISSUES ADDRESSED:
- None



## TESTS PERFORMED:
- Applied SQL
- Checked all vendors (Horde & Alliance)



## HOW TO TEST THE CHANGES:



1. Add the following achievements so you are able to test HORDE site (.achievement add xxxx):
- 2783
- 2784
- 2785
- 2786
- 2787

2. Add the following achievements so you are able to test ALLIANCE site (.achievement add xxxx):

- 2777
- 2778
- 2779
- 2780
- 2781

3. .go 8414 662 550 571 and check all the horde vendors (All should sell in the same sequence and no items are duplicated)

4. .go 8613 659 550 571 and check all the alliance vendors (All should sell in the same sequence and no items are duplicated)

## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
